### ISSUE TITLE
[HxMessageBox] MessageBoxRequest doesn't contain options for Button Settings

### ISSUE THREAD

**Issue by surgicalcoder** [havit/Havit.Blazor#606]
As title says, `MessageBoxRequest` is missing parameters for `PrimaryButtonSettings `/ `SecondaryButtonSettings`, I think these are the only two parameters of `<HxMessageBox` that are missing from `MessageBoxRequest`

**Comment by hakenr** [comment]
You're right! We'd greatly appreciate it if you could submit a pull request for this enhancement.

In the meantime, you can utilize the `MessageBoxRequest.AdditionalAttributes` property to pass any required parameters.
